flink-connector-mysql-cdc/src/main/java/com/ververica/cdc/connectors/mysql/source/reader/MySqlSourceReader.java

```diff
@@ -338,6 +341,19 @@ private void fillMetaDataForBinlogSplit(BinlogSplitMetaEvent metadataEvent) {
         }
     }
 
+    private void logCurrentBinlogOffsets(List<MySqlSplit> splits, long checkpointId) {
+        if (!LOG.isInfoEnabled()) {
+            return;
+        }
+        for (MySqlSplit split : splits) {
+            if (!split.isBinlogSplit()) {
+                return;
+            }
+            BinlogOffset offset = split.asBinlogSplit().getStartingOffset();
+            LOG.info("Binlog offset on checkpoint {}: {}", checkpointId, offset);
```

Comment:
Add some tips that user can use specific offset way to read the rest binglog after this offset,  the feamework can offer exactly-once semantics on this case.

Reply:
please add this case to connector docs as well, this is the most important value the PR brings.